trustfall_core/src/schema/error.rs

```diff
@@ -125,6 +125,9 @@ pub enum InvalidSchemaError {
         Only interfaces can be implemented by other types."
     )]
     ImplementingNonInterface(String, String),
+
+    #[error("Type or Interface \"{0}\" defines the field \"{1}\" multiple times.")]
```

Comment:
```suggestion
    #[error("Type \"{0}\" defines the field \"{1}\" multiple times.")]
```
GraphQL terminology (and by extension Trustfall terminology) is a mess here, but all vertex types are types, and `type` in GraphQL is usually called "object type" which is a term I don't like but I don't have a better one. In any case, including both terms here is probably overall more confusing, so let's stick to the simpler case.